src/phoenix/datasets/evaluators/code_evaluators.py

```diff
@@ -32,18 +21,14 @@ def evaluate(self, exp_run: ExperimentRun, example: Example) -> EvaluationResult
         )
 
 
-class ContainsKeyword:
-    annotator_kind = AnnotatorKind.CODE.value
-
+class ContainsKeyword(Evaluator):
     def __init__(self, keyword: str, name: Optional[str] = None) -> None:
         self.keyword = keyword
-        self.name = name or f"Contains({repr(keyword)})"
+        self._name = name or f"Contains({repr(keyword)})"
 
-    def evaluate(self, exp_run: ExperimentRun, example: Example) -> EvaluationResult:
-        assert exp_run.output is not None
-        result = _unwrap_json(exp_run.output.result)
-        assert isinstance(result, str), "Experiment run output must be a string"
-        found = self.keyword in result
+    def evaluate(self, *, output: Optional[TaskOutput] = None, **_: Any) -> EvaluationResult:
+        assert isinstance(output, str), "Experiment run output must be a string"
```

Comment:
I wonder if this is too restrictive - this will end up causing it to error out post task-run which at that point is too late and also maybe too restrictive. 

Since the output is serializable what if we serialize it to a string? 

On the flip side I'm starting to feel these pre-baked code evals might be a tad too simple for them to be valuable over a simple string